### DISCUSSION TITLE
CSS Styling

### DISCUSSION THREAD

**Discussion by smartqasa** [punxaphil/custom-sonos-card#148] [Q&A]
How might I change the font-size and font-weight of the title with the new V6 card?

I have tried this: and various iterations with no success.

    card_mod:
      sytle: |
        ha-card {
          font-size: 22px !important;
        }

**Comment by punxaphil** [comment]
It's documented in card_mod how to do it. https://github.com/thomasloven/lovelace-card-mod#dom-navigation
```
card_mod:
  style: |
      div {
        font-size: 22px !important;
      }
```

**Reply by smartqasa** [reply]
Thanks. I really appreciate the quick response and I love the new card. Great work!!

I use card_mod extensively and am familiar with documentation. "div" doesn't work. I will figure it out.

**Comment by punxaphil** [comment]
It works:
https://github.com/johanfrick/custom-sonos-card/assets/26582898/e0de06e9-5602-4310-aa8e-63ee7d1ce423

**Comment by punxaphil** [comment]
Not sure if you noticed, but you have a typo (sytle)
> 
> ```
> card_mod:
>   sytle: |
>     ha-card {
> ```

**Comment by smartqasa** [comment]
Did catch the type-o. Yes, it works on title. My bad for explaining improperly. I'm trying to change the font-size (and font-weight) of the header (sonos-player-header). 

I redownloaded v6.3; restarted HA (Home Assistant 2023.6.3  Supervisor 2023.06.2  Operating System 10.3  Frontend 20230608.0 - latest) and cleared cache on Chrome.

https://photos.app.goo.gl/7kHzMHc3P4jGpZpy7

Again, thanks so much for the quick responses.

**Comment by smartqasa** [comment]
I decided to use the "title" after all. However, using the title creates an overflow (and scroll bar). 

https://photos.app.goo.gl/gfYjrAAi2f3SWMM69

Is there a way to expand the card to eliminate the scroll?
